src/mudlet.pro

```diff
@@ -102,7 +102,7 @@ GIT_EXECUTABLE = git
 GIT_SHA1 = $$system($$GIT_EXECUTABLE rev-parse --short HEAD)
 
 # Use the result in your QMake project
-message("Git SHA1: " $$GIT_SHA1)
+!build_pass:message("Git SHA1: " $${GIT_SHA1})
```

Comment:
The `!build_pass` means that this now only gets reported once for Windows builds (where 2 or 3 passes are make through the file - this limits the message to only be shown during the first run that makes the top-level `MakeFile` and not the following `MakeFile.release` and/or `MakeFile.debug` files)...